sarsen/radiometry.py

```diff
@@ -172,7 +174,7 @@ def gamma_weights_nearest(
 
 
 def azimuth_slant_range_grid(
-    measurement_ds: xr.Dataset,
+    measurement_ds: xr.DataArray,
```

Comment:
@alexamici looks like this should be a DataArray but the variable name is confusing

Reply:
D'oh! I'll change it outside of this PR